tracing-test/README.md

```diff
@@ -0,0 +1,43 @@
+# tracing-test
+
+Testing utilities for the tracing ecosystem
+
+[![Crates.io][crates-badge]][crates-url]
+[![Documentation][docs-badge]][docs-url]
+[![Documentation (master)][docs-master-badge]][docs-master-url]
+[![MIT licensed][mit-badge]][mit-url]
+[![Build Status][actions-badge]][actions-url]
+[![Discord chat][discord-badge]][discord-url]
+
+[Documentation][docs-url] | [Chat][discord-url]
+
+[crates-badge]: https://img.shields.io/crates/v/tracing.svg
+[crates-url]: https://crates.io/crates/tracing/0.1.15
+[docs-badge]: https://docs.rs/tracing/badge.svg
+[docs-url]: https://docs.rs/tracing/0.1.15
+[docs-master-badge]: https://img.shields.io/badge/docs-master-blue
+[docs-master-url]: https://tracing-rs.netlify.com/tracing
+[mit-badge]: https://img.shields.io/badge/license-MIT-blue.svg
+[mit-url]: LICENSE
+[actions-badge]: https://github.com/tokio-rs/tracing/workflows/CI/badge.svg
+[actions-url]:https://github.com/tokio-rs/tracing/actions?query=workflow%3ACI
+[discord-badge]: https://img.shields.io/discord/500028886025895936?logo=discord&label=discord&logoColor=white
+[discord-url]: https://discord.gg/EeF3cQw
+
+
+## Overview
+
+`tracing-test` provides testing utilities that are used internally in the
+`tracing` codebase and may be useful elsewhere in the `tracing` ecosystem.
+
+This crate is unstable and does not make any stability guarantees.
+
+## License
+
+This project is licensed under the [MIT license](LICENSE).
```

Comment:
Let's also add a copy of the MIT license in the `tracing-test` dir. Since `cargo publish` is run in that directory, it will only include a copy of the license if we add one there (see https://github.com/tokio-rs/tracing/pull/842).